.sync.yml

```diff
@@ -15,10 +15,14 @@ Gemfile:
       - gem: puppet-blacksmith
         version: '~> 3.4'
 
-# For the moment don't do any rubocop checks.
+.gitignore:
+  required:
+    - ---.project
+
 .rubocop.yml:
-  include_todos: true
-  selected_profile: off
+    require:
```

Comment:
These requires are not needed as rubocop-i18n and rubocop-rspec are included by default.